### DISCUSSION TITLE
如何设置一个频道号在列表中只显示一次，比如cctv1 左侧频道树只显示一个cctv1

### DISCUSSION THREAD

**Discussion by CharlesJay01** [Guovin/iptv-api#756] [Q&A] [answered]
![微信图片_20241228153222](https://github.com/user-attachments/assets/1589269b-8932-4f22-85cf-a59df8267d68)

**Comment by piaoliuchong** [comment]
> ![微信图片_20241228153222](https://github.com/user-attachments/assets/1589269b-8932-4f22-85cf-a59df8267d68)
> 

这跟你的播放器有关吧。

**Comment by Guovin** [comment, marked as answer]
这是播放器行为，如果切换txt结果无法解决，建议更换播放器
